index.html

```diff
@@ -4,7 +4,19 @@
     <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
     <meta charset="utf-8">
     <meta content="width=device-width, initial-scale=1.0" name="viewport">
+
+    <!-- Search engine optimisatin -->
+    <meta name="description" content="Delhi LUG Website">
+    <meta name="keywords" content=" Linux, Linux User Group, LUG, DELHI LUG, Linux Assistance">
+    <meta name="author" content="ILUGD">
+    <!-- End Search Engine Optimisation -->
+
     <title>Indian Linux User Group Delhi</title>
+
+ <!-- Site favicon -->
```

Comment:
Why is this not indented like the rest of the file?

Reply:
sorry, my mistake, I focused on work rather than viewing indention, okay lemme edit it.

Reply:
@Manas-kashyap Did you edit it further,  from that point?